packages/expo-updates/ios/Tests/EXUpdatesFileDownloaderManifestParsingTest.m

```diff
@@ -112,4 +179,38 @@ - (void)testManifestParsing_MultipartBody
   XCTAssertNotNil(resultUpdateManifest);
 }
 
+- (void)testManifestParsing_JSONBodySigned_UnsignedRequest {
```

Comment:
```suggestion
- (void)testManifestParsing_JSONBodySigned_UnsignedRequest {
```
test name is a bit confusing - the request wants a signature from the server but the response is unsigned